drift-codec-utils/src/main/java/com/facebook/drift/codec/utils/DataSizeToBytesThriftCodec.java

```diff
@@ -42,7 +42,7 @@ public DataSizeToBytesThriftCodec(ThriftCatalog thriftCatalog)
     }
 
     @CodecThriftType
-    public static ThriftType getThriftType()
+    public static ThriftType getThriftType(ThriftCatalog catalog)
```

Comment:
Could you please explain what problem are you trying to solve? Why is it necessary to have access to `ThriftCatalog` to define a thrift type? Ideally this method should be pure constant / static.